Server/Terrain.py

```diff
@@ -0,0 +1,91 @@
+import random as ra
+import math as mat
+import bresenham as br
+
+
+SCREEN_WIDTH = 800
+SCREEN_HEIGHT = 600
+
+BASE_LEVEL = 120
+MIN_LEVEL = 50
+
+MAX_ABS_ANGLE = 15
+NO_ANGLE_BASE_P = 0.45
```

Comment:
Don't put code outside of classes and funcs only if you realy need to do so! Class variables/constants can be definde like:
class Domi:
....SZUPER = True
....     IQ = 200
   .... def __init__(self):
        ............if self.SZUPER:  # class variables can be accessed by cls as well
      ............      print("Éljen Domi")